src/main/java/com/commercetools/sync/products/utils/ProductUpdateActionUtils.java

```diff
@@ -441,6 +450,36 @@
         return allVariants;
     }
 
+    private static boolean filterDuplicateSameForAllAction(
```

Comment:
Why not name parameter `updateActions` the same way as it is passed to the method: `sameForAllUpdateActions`? Also, the method returns a boolean - maybe then we should name it accordingly i.e.: `hasDuplicateSameForAllAction`?

Reply:
In general, it feels like duplicate sameForAll actions should be omitted here (https://github.com/commercetools/commercetools-sync-java/pull/508/files#diff-540361586e271c4157d1afe06f9c8636R494) instead of first generating them and later filtering them out?

Reply:
I'm sorry the link provided doesn't give me a clue as it opens the pull request again